cpp/include/cudf_test/column_wrapper.hpp

```diff
@@ -103,7 +111,15 @@ struct fixed_width_type_converter {
     return element;
   }
 
-  // Are the types convertible or can target be constructed from source?
+  /// Are the types convertible or can target be constructed from source?
+  /**
+   * @brief  Convert types if possible, otherwise construct target from source.
+   *
+   * @tparam FromT Source type
+   * @tparam ToT Target type
+   * @param element Source value
+   * @return The converted target value
+   */
```

Comment:
```suggestion
  /**
   * @brief Convert types if possible, otherwise construct target from source.
   *
   * @tparam FromT Source type
   * @tparam ToT Target type
   * @param element Source value
   * @return The converted target value
   */
```